cpg_workflows/defaults.toml

```diff
@@ -105,6 +105,9 @@ delete_scratch_on_exit = false
 # defaults in code are 40 for genomes, none for exomes
 #haplotypecaller_storage = 80
 
+# Use highmem machine type for alignment step
+align_use_highmem = false
```

Comment:
I guess with `.get()`, this could be commented-out-by-default too, like the other resource_overrides entries.